tutorcredentials/__about__.py

```diff
@@ -1 +1 @@
-__version__ = "17.0.0"
+__version__ = "17.0.1"
```

Comment:
You, my friend, forgot to collect changelog entries ;) https://github.com/overhangio/tutor/pull/991/files#diff-f94b76ecc373d484f1bf8b218e690f92be03225e3a5940f6f0d392b816783901

Reply:
Actually, there were no changelog entries created initially, No changelog fragments to collect.

Reply:
Good point! There should have been an entry in this PR though: https://github.com/overhangio/tutor-credentials/pull/31

Reply:
forgot to comment on latest PR that is merged for the changelog entries.

Reply:
yup!

Reply:
It's OK. I just pushed a changelog entry to master.

Reply:
thanks regis.